docs/api/translator.md

```
@@ -0,0 +1,88 @@
+# Translator
+
+The translator is a core component used to translate texts between languages.
+
+The translator API is designed to handle texts efficiently in terms of the ratio of texts translated per time to number of requests.
+
+Translator methods may be called intensively, which is why it is important to set correct rate limits.
+
+### Translator API
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)